code/Modules/Items/Carriables/Carriable.cs

```diff
@@ -9,6 +9,7 @@ public override void Spawn()
 		{
 			base.Spawn();
 
+			SetupPhysicsFromModel(PhysicsMotionType.Dynamic);
```

Comment:
suggestin: comment

Reply:
I don't know exactly what SetupPhysicsFromModel does for the moment, only that it is required so that Rubikon know the collision group